pkg/codegen/python/gen.go

```diff
@@ -760,7 +770,34 @@ func (mod *modContext) genConfig(variables []*schema.Property) (string, error) {
 			configFetch += " or " + v
 		}
 
-		fmt.Fprintf(w, "%s = %s\n", PyName(p.Name), configFetch)
+		fmt.Fprintf(w, "%s@property\n", indent)
+		fmt.Fprintf(w, "%sdef %s(self) -> %s:\n", indent, PyName(p.Name), mod.typeString(p.Type, false, true))
```

Comment:
Shouldn't the type always be `str` (or `Optional[str]`)? Also, it might be worth adding a brief comment on why its typed as `str` (for historical reasons to maintain backcompat).

Reply:
whoopsies